radio/src/targets/taranis/lcd_driver_spi.cpp

```diff
@@ -111,7 +111,11 @@ void lcdStart()
 #if defined(LCD_VERTICAL_INVERT)
   // T12 and TX12 have the screen inverted.
   lcdWriteCommand(0xe2); // (14) Soft reset
+#if defined(RADIO_LR3PRO)
+  lcdWriteCommand(0xa1); // Set seg
```

Comment:
What is this? We need it more generically as.

Reply:
![unknown](https://user-images.githubusercontent.com/54450502/174591063-50c70099-0398-444f-9a5e-6a2a1d0e89ed.png)
The setting here is to make OLEDs mirror left and right.

Reply:
Better for this to be generic define... i.e. you can see there is a LCD_VERTICAL_INVERT... is this perhaps `LCD_HORIZONTAL_INVERT`?